includes/abstracts/abstract-wc-legacy-product.php

```diff
@@ -129,6 +129,7 @@ public function get_post_data() {
 	 * @return string
 	 */
 	public function get_title() {
```

Comment:
No need to adjust anything yet, but we may want to keep something like this around within the classes for things like simple products which append the parent grouped product name to the title before display. Unless we write a template function to replace this?